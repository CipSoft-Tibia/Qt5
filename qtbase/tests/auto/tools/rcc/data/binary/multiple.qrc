<!DOCTYPE RCC><RCC version="1.0">
    <qresource>
    <file>blahblah.txt</file>
    <file>currentdir.txt</file>
    <file>currentdir2.txt</file>
    </qresource>
    <qresource><file>subdir/subdir.txt</file></qresource>
    <qresource><file>otherdir/</file>
    </qresource>
</RCC>
