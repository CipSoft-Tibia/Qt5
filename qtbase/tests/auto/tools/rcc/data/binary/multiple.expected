blahblah.txt blahblah.txt
currentdir.txt currentdir.txt
currentdir2.txt currentdir2.txt
subdir/subdir.txt subdir/subdir.txt
otherdir/otherdir.txt otherdir/otherdir.txt
